src/Handler/Table/Preview/PreviewTableHandler.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Teradata\Handler\Table\Preview;
+
+use Google\Protobuf\Internal\GPBType;
+use Google\Protobuf\Internal\Message;
+use Google\Protobuf\Internal\RepeatedField;
+use Google\Protobuf\NullValue;
+use Google\Protobuf\Value;
+use Keboola\Datatype\Definition\BaseType;
+use Keboola\Datatype\Definition\Teradata;
+use Keboola\StorageDriver\Command\Table\PreviewTableCommand;
+use Keboola\StorageDriver\Command\Table\PreviewTableResponse;
+use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\Teradata\TeradataSessionManager;
+use Keboola\TableBackendUtils\Column\Teradata\TeradataColumn;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+use Keboola\TableBackendUtils\Table\Teradata\TeradataTableReflection;
+
+class PreviewTableHandler implements DriverCommandHandlerInterface
+{
+    public const STRING_MAX_LENGTH = 50;
+
+    private TeradataSessionManager $manager;
+
+    public function __construct(TeradataSessionManager $manager)
+    {
+        $this->manager = $manager;
+    }
+
+    /**
+     * @inheritDoc
+     * @param GenericBackendCredentials $credentials
+     * @param PreviewTableCommand $command
+     */
+    public function __invoke(
+        Message $credentials,
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof PreviewTableCommand);
+
+        // validate
+        assert($command->getPath()->count() === 1, 'CreateTableCommand.path is required and size must equal 1');
```

Comment:
tohle zavani nasazenim nejakeho validatoru, neco a la RequestObject co bude mit pravidla a zvaliduje vstup. Skoda ze nam tom nijak nepomuzou ty protobuffery

Reply:
dohodli jsme se, ze je to takhle zatim dostacujici